backend/director/llm/anthropic.py

```
@@ -157,7 +157,7 @@ def chat_completions(
         try:
             response = self.client.messages.create(**params)
         except Exception as e:
-            raise e
+            logger.exception("Anthropic LLM call failed")
```

Ruff (0.15.10)

[error] Undefined name `logger`

(F821)